### PR TITLE
Better issue templates

### PR DESCRIPTION
**Pull request by cloudcloudskies**
Updated issue templates to make them look like Vesktop's issue templates.

### CODE REVIEW

**Review by ryan-0324 — CHANGES_REQUESTED** [review]
We're is the contraction of we are.

File: .github/ISSUE_TEMPLATE/blank.yml [hunk 1]
```diff
@@ -1,30 +1,30 @@
 name: Blank Issue
-description: Create a blank issue. ALWAYS FIRST USE OUR SUPPORT CHANNEL! ONLY USE THIS FORM IF YOU ARE A CONTRIBUTOR OR WERE TOLD TO DO SO IN THE SUPPORT CHANNEL.
+description: Create a blank issue. Always first use our support channel! Only use this form if you're a contributor or we're told to do so in the support channel.
```
Comment (ryan-0324) [inline .github/ISSUE_TEMPLATE/blank.yml:2]:
```suggestion
description: Create a blank issue. Always first use our support channel! Only use this form if you're a contributor or were told to do so in the support channel.
```

File: .github/ISSUE_TEMPLATE/blank.yml [hunk 2]
```diff
@@ -1,30 +1,30 @@
...
-        
-        DO NOT USE THIS FORM, unless
-        - you are a vencord contributor
-        - you were given explicit permission to use this form by a moderator in our support server
-        - you are filing a security related report
+          This form is ONLY FOR DEVELOPERS. Your issue will be closed and you will be possibly blocked from the repo if you ignore this.
+
+          Don't use this form, unless:
+          - You're a Vencord contributor,
+          - You we're given explicit permission to use this form by a moderator in our support server,
```
Comment (ryan-0324) [inline .github/ISSUE_TEMPLATE/blank.yml:14]:
```suggestion
          - You were given explicit permission to use this form by a moderator in our support server,
```

File: .github/ISSUE_TEMPLATE/bug_report.yml [hunk 3]
```diff
@@ -1,81 +1,81 @@
-name: Bug/Crash Report
-description: Create a bug or crash report for Vencord. ALWAYS FIRST USE OUR SUPPORT CHANNEL! ONLY USE THIS FORM IF YOU ARE A CONTRIBUTOR OR WERE TOLD TO DO SO IN THE SUPPORT CHANNEL.
-labels: [bug]
+name: 🐛 Bug / Crash Report
+description: Create a bug or crash report for Vesktop. Always use our support channel! Only use this form if you're a contributor or we're told to do so in the support channel.
```
Comment (ryan-0324) [inline .github/ISSUE_TEMPLATE/bug_report.yml:2]:
````suggestion
description: Create a bug or crash report for Vesktop. Always use our support channel! Only use this form if you're a contributor or were told to do so in the support channel.
```

File: .github/ISSUE_TEMPLATE/bug_report.yml [hunk 4]
```diff
@@ -1,81 +1,81 @@
...
-        DO NOT USE THIS FORM, unless
-        - you are a vencord contributor
-        - you were given explicit permission to use this form by a moderator in our support server
-        
-        DO NOT USE THIS FORM FOR SECURITY RELATED ISSUES. [CREATE A SECURITY ADVISORY INSTEAD.](https://github.com/Vendicated/Vencord/security/advisories/new)
+          This form is ONLY FOR DEVELOPERS. Your issue will be closed and you will be possibly blocked from the repo if you ignore this.
+
+          Please don't use this form, unless:
+          - You're a Vencord contributor,
+          - You we're given explicit permission to use this form by a moderator in our support server.
```
Comment (ryan-0324) [inline .github/ISSUE_TEMPLATE/bug_report.yml:16]:
```suggestion
          - You were given explicit permission to use this form by a moderator in our support server.
```

**Review by ryan-0324 — CHANGES_REQUESTED** [review]
Sorry, I submitted the last review wrong.

File: .github/ISSUE_TEMPLATE/bug_report.yml (hunk 3 above)
Comment (schoeneBiene) [inline .github/ISSUE_TEMPLATE/bug_report.yml:2]:
```suggestion
description: Create a bug or crash report for Vencord. Always use our support channel! Only use this form if you're a contributor or we're told to do so in the support channel.
```

**Comment by Vendicated** [comment]
they're fine as is for now. i might overhaul them from scratch myself in the future

thanks regardless

**Pull request closed without merging**